### ISSUE TITLE
Refactor so everything works locally

### ISSUE THREAD

**Issue by matt-dray** [matt-dray/tamRgo#20]
Reconsider use of gists/remote blueprint storage, i.e. all the work done to date (2022-09-19 1800ish).

New process would be entirely local, like:

1. `lay_egg()` to create blueprint as list of lists (including unchangeable unique pet ID value) and write to cache as an rds (i.e. building the whole package around #5)
2. Interact with pet/view stats (and potentially on package load as per #19)
    - Flow like #18, but with the cached blueprint rather than the pet/gist ID
    - Would first check for cached blueprint (#14) and tell user to `lay_egg()` if it doesn't exist
    - If blueprint exists in cache, read it, record datetime of interaction, update values as required since last interaction (e.g. update the hunger value by 2 if hunger increases by 1 per day and it's been 2 days since the last interaction), and update status values if the interaction is one that changes a status value (e.g. if interaction is to feed pet, then reduce hunger value by 1)

Consider:

* Keeping the data in the cache would mean it's not in the global environment for the session, minimsing the chance of 'user interference' (#11)
* That the approach wouldn't require any internet connection (so #15 and #16 would be redundant in the short term), but could be expanded to include gists later
* User could ultimately choose to push the blueprint to a gist, which would be useful later for battling purposes (#8) and so they could interact with the pet on multiple devices

Basically, start with local only and develop it from there.

**Comment by matt-dray** [comment]
[R-hub blog](https://blog.r-hub.io/2020/03/12/user-preferences/) and [{rappdirs}](https://rappdirs.r-lib.org/) still relevant re cache.
